_posts/2020-10-01-mike-spencer.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Pushing the bubble"
+author: "@pweschmidt"
+rse: "Mike Spencer"
+phenotype: 
+excerpt: "From assessing snow fall in Britain to livestock movements, Mike has provided his expertise in data science to policy makers and influencers. So testing the data and underlying assumptions is an essential step. For this it is sometimes necessary to move outside our comfort zone."
+date: 2020-10-02 8:30:00
```

Comment:
I think this Thursday is October 1st?
```suggestion
date: 2020-10-01 8:30:00
```

Reply:
oops - erm, yes. So it is. Sorry, got it mixed up in the calendar